src/misc/person.v1.yaml

```diff
@@ -26,12 +26,37 @@ properties:
                       - Schekman, Randy
                       - VijayRaghavan, Krishnaswamy
                       - Li Wenhui
+            given_names:
+                title: Given names
+                description: |
+                    The given names of the person.
+
+                    For example:
+                      - Randy
+            surname:
```

Comment:
We originally went for just single strings as names are a _very_ thorny issue.

(Caveat to the following: might be completely wrong.) I believe one of our senior editors has a name in the format `[Father's name] [Given name]`, which looks to get rather mangled (including in our own JATS).

Unfortunately we're dealing with imperfect inputs and outputs. I'm tempted to say we should try and avoid componentising it here (since we can't do it properly) and instead just have 'first names' and 'last names' (with the latter optional). Or a list of 1 or 2 values. Makes it ambiguous as to what these names are, but that's the point.

Reply:
Yeah. This has been an ongoing issue with eJP, apparently they need to have the `surname`  to reconcile users, and they use the JATS one. The idea is that we don't replace the existing `index` and `preferred` but complement with fields for given name and surname ( that staff can fill out in Journal)

Reply:
@thewilkybarkid ... are you happy if I resolve this conversation?

Reply:
I'll never be happy with it, but yes. 😉